### PR TITLE
board/calliope-mini: use openocd as default flasher and rm fscopy

### PR DESCRIPTION
**Pull request by haukepetersen**
### Contribution description
The `fscopy` flash script is quite hacky and has never been working very well for most people. Also, `openocd` is now nowadays working just fine with the `calliope-mini`. So there is IMHO no reason to stick with the shaky `fscopy` procedure, and as proposed in this PR I even opt for removing that option completely, decreasing maintenance effort...

### Testing procedure
Simply flash the calliope with any example of your choice. The build system should use `openocd` as default option.

### CODE REVIEW

**Comment by SemjonKerner** [comment]
I don't have a strong opinion.. Since its only used by microbit and calliope it is obviously an option to delete the flasher. I think maintenance is the best reason to drop this script.

File: boards/calliope-mini/dist/flash.sh [hunk 1]
```diff
@@ -1,67 +0,0 @@
-#!/bin/sh
```
Comment (SemjonKerner) [inline boards/calliope-mini/dist/flash.sh]: I prefer you split the commits

**Comment by haukepetersen** [comment]
changes are now in two commits

**Comment by smlng** [comment]
@SemjonKerner all good to go here?

**Comment by cladmi** [comment]
Is there a global movement to remove the `fscopy` scripts?
If it is so, then the `mbed_lpc1768` could also need a migration to openocd.

**Comment by SemjonKerner** [comment]
@smlng @haukepetersen thats how i pictured it. Didn't test, but looks good.

@cladmi Is there a reason to keep fscopy, when all boards in question are supported by openocd?

**Comment by smlng** [comment]
@cladmi: to me it sound reasonable to reduce the number of extra tools around RIOT, especially when openocd does the same job just fine: go for it - and that said: yes, lets remove fscopy dependency from other boards if possible.

**Comment by smlng** [comment]
@SemjonKerner I tested and it (still) works, mind to ACK (or dismiss review)?

**Comment by cladmi** [comment]
In case of a global cleanup it would be good to do it with a tracking issue at the beginning, so no boards get forgotten.

I got the board, I will see if I manage to write/adapt the openocd configuration, I never did it.

**Comment by miri64** [comment]
I'd say it isn't a global cleanup. It's just replacing an unreliable flashing tool (but easy to port, and thus maybe provided as a first entry step for a new board) for these kind of platforms with a reliable one. We don't want to make ourselves too dependable on one tool if that tool does not support a new shiney board yet and all we have is their default way of flashing (i.e. drag-and-dropping binaries into a filesystem).

**Comment by cladmi** [comment]
Removing 2 out of the 3 uses of this method, yes it should be a cleanup.

**Comment by aabadie** [comment]
> In case of a global cleanup it would be good to do it with a tracking issue at the beginning, so no boards get forgotten.

There's no need for a tracking issue for this, since we can't consider this as a global cleanup. It this particular case, it would add more work than it saves. microbit and calliope-mini are done. Only mbed-lpc1768 remains. So that's ok.

**Comment by miri64** [comment]
> Removing 2 out of the 3 uses of this method, yes it should be a cleanup.

Cleanup would imply that we don't accept new boards using this method. IMHO this is overkill and not constructive.

**Pull request merged**